src/OrbitGl/TimeGraph.cpp

```diff
@@ -60,15 +60,19 @@ TimeGraph::TimeGraph(AccessibleInterfaceProvider* parent, OrbitApp* app,
     : orbit_gl::CaptureViewElement(nullptr, viewport, &layout_),
       accessible_parent_{parent},
       batcher_(BatcherId::kTimeGraph),
-      manual_instrumentation_manager_{app->GetManualInstrumentationManager()},
       thread_track_data_provider_(capture_data->GetThreadTrackDataProvider()),
       capture_data_{capture_data},
       app_{app} {
-  text_renderer_static_.Init();
+  if (app_ != nullptr) {
```

Comment:
Do we really want to allow `app_` being null? Feels a bit strange. And for testing, we should always be able to fake/mock it, right?

Reply:
> Thanks for this Florian. I left a few comments.
> 
> I was thinking that these test should be in TimeGraph instead now that we are moving the sliders there. What do you think about that? Should we merge this and move them in the next weeks?

No, I would argue exactly the other way around. I want to have them now because I want to make sure that subsequent refactorings will not break the functionality. I've used those to make sure that moving the scroll wheel handling into `TimeGraph` does not change the functionality expected from `CaptureWindow`.

They probably need to be adapted slightly with your refactorings depending on whether `CaptureWindow` still has access to the sliders, but that's ok - these tests can also work by just verifying the min / max timestamps and the track positions, and you can move the parts that tests the sliders into `TimeGraph`. But we hand of the handling of mouse events from Qt to `CaptureWindow`, so I want the integration tests for this to start at this level.

Reply:
No, we don't, and I'm aware this is not ideal. But mocking app is still an incredible pain and effort, and I have the choice between this or no Unit Tests for the time being :-)

Reply:
But can't we simply define "null-like" mock of App once and use it in different tests? (Please note, that I will not insist on this for this PR).

Reply:
I'll open a bug for this to tackle it in the future - I don't think I'll have time to address it this week.

Reply:
b/226376310 - I didn't add a TODO into the code as this is distributed throughout the code base.